receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -76,6 +76,8 @@ type transaction struct {
 	externalLabels       labels.Labels
 	logger               *zap.Logger
 	obsrecv              *obsreport.Receiver
+	stalenessStore       *stalenessStore
+	startTime            int64
```

Comment:
What is the unit of time? Is this seconds, nanoseconds? Since when? Perhaps use time.Time or a different stronger type?

Reply:
No need for a stronger type given the internal type that Prometheus uses. I'll modify the variable name to startTimeMs.